projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts

```diff
@@ -8,7 +8,8 @@ import * as _ from 'lodash-es';
 @Component({
   selector: 'lib-editor-base',
   templateUrl: './editor-base.component.html',
-  styleUrls: ['./editor-base.component.scss']
+  styleUrls: ['./editor-base.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Why `ViewEncapsulation.none`. then the styles won't be scope to this particular component no?

Reply:
Yes, we need css to be reuse in other components